src/coreclr/vm/prestub.cpp

```diff
@@ -1910,6 +1925,11 @@ extern "C" PCODE STDCALL PreStubWorker(TransitionBlock* pTransitionBlock, Method
 {
     PCODE pbRetVal = NULL;
 
+    Module* pModule = NULL;
+    FrameWithCookie<ExternalMethodFrame> frame(pTransitionBlock);
+    ExternalMethodFrame * pEMFrame = &frame;
+    pModule = ExecutionManager::FindReadyToRunModule((TADDR)(((BYTE*)pEMFrame->GetReturnAddress())-1));
```

Comment:
This is again a linear search through all loaded assemblies before JITing every method.